pkg/api/controllers/account_controller_test.go

```diff
@@ -144,6 +144,111 @@ func TestGetAccounts(t *testing.T) {
 				assert.Len(t, resp.Cursor.Data, 1)
 				assert.Equal(t, resp.Cursor.Data[0].Address, "bob")
 
+				// test filter by balance >= 50 with default operator
+				{
```

Comment:
Instead of brackets, I would suggest to use the `t.Run()` method to split in named sub tests, just as I did in this PR: https://github.com/numary/ledger/pull/218

Reply:
you're right, it's better this way, i'll fix it